source/includes/kubernetes/_k8_secrets.md

```diff
@@ -0,0 +1,90 @@
+## Secrets
+
+<!-------------------- LIST Secrets -------------------->
+
+### List secrets
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/secrets"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "id": "default-token-xxxmt/default",
+      "apiVersion": "v1",
+      "data": {
+            "ca.crt": [],
+            "namespace": [],
+            "token": []
+      },
+      "kind": "Secret",
+      "metadata": {},
+      "type": "kubernetes.io/service-account-token"
+    }
+  ],
+  "metadata": {
+    "recordCount": 100
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/secrets</code>
+
+Retrieve a list of all secrets in a given [environment](#administration-environments).
+
+| Attributes                 | &nbsp;                                                                        |
+| -------------------------- | ----------------------------------------------------------------------------- |
+| `id` <br/>_string_         | The id of the secret                                                          |
+| `apiVersion` <br/>_string_ | The API version used to retrieve the secret                                   |
+| `data` <br/>_string_       | Data contains the secret data.                                                |
```

Comment:
This sentence is inconsistent from the rest of the entries. I would suggest it be changed to "The data that the secret contains"